src/freenet/node/Node.java

```diff
@@ -4923,4 +5182,161 @@ public NodeDiagnostics getNodeDiagnostics() {
     public boolean isNodeDiagnosticsEnabled() {
         return enableNodeDiagnostics;
     }
+
+    public NodeStats getNodeStats() {
+        return nodeStats;
+    }
+
+    public PersistentConfig getConfig() {
+        return config;
+    }
+
+    public NodeGetPubkey getGetPubKey() {
+        return getPubKey;
+    }
+
+    public NodeIPDetector getIpDetector() {
+        return ipDetector;
+    }
+
+    public boolean isDisableProbabilisticHTLs() {
+        return disableProbabilisticHTLs;
+    }
+
+    public RequestTracker getTracker() {
+        return tracker;
+    }
+
+    public PeerManager getPeers() {
+        return peers;
+    }
+
+    public RandomSource getRandom() {
+        return random;
+    }
+
+    public SecureRandom getSecureRandom() {
+        return secureRandom;
+    }
+
+    public Random getFastWeakRandom() {
+        return fastWeakRandom;
+    }
+
+    public NodeCrypto getDarknetCrypto() {
+        return darknetCrypto;
+    }
+
+    public Executor getExecutor() {
+        return executor;
+    }
+
+    public PacketSender getPacketSender() {
+        return ps;
```

Comment:
that makes code much easier to follow — thank you!

Reply:
In this particular instance it does, yes… most of it is boilerplate, though.